From 3b64d9b087a9347f835fee8751d28870667d528a Mon Sep 17 00:00:00 2001
From: Justin Maggard <jmaggard10@gmail.com>
Date: Wed, 9 Apr 2014 15:51:55 -0700
Subject: [PATCH] Revert "src: Let systemd unmount remote filesystems before
 stopping connman"

This reverts commit 996cbf10bad3b5ecde471596299fb9e1e1726dc5.

This change caused a systemd loop, which was reported as
"Found ordering cycle on basic.target/start", and ended up killing off
dbus.socket and avahi-daemon.socket to break the ordering cycle.
---
 src/connman.service.in |    1 -
 1 file changed, 1 deletion(-)

diff --git a/src/connman.service.in b/src/connman.service.in
index 7b6195e..cfac76c 100644
--- a/src/connman.service.in
+++ b/src/connman.service.in
@@ -2,7 +2,6 @@
 Description=Connection service
 Requires=dbus.socket
 After=dbus.socket
-Before=remote-fs.target
 
 [Service]
 Type=dbus
